defaults/property-defaults.yaml

```diff
@@ -7,6 +7,22 @@ defaults:
       quality_impressions_per_duration_s: 0.100000000
       revenue_allocation_to_digital_pct: 100
       revenue_allocation_to_display_pct: 14.900000000
+  display-app:
```

Comment:
since this file is auto generated, I think it'd be good to add a comment to the top of the file to make clear that it's auto generated